src/Symfony/Bundle/FrameworkBundle/FrameworkBundle.php

```diff
@@ -117,6 +118,7 @@ public function build(ContainerBuilder $container)
         $container->addCompilerPass(new CachePoolPrunerPass(), PassConfig::TYPE_AFTER_REMOVING);
         $this->addCompilerPassIfExists($container, FormPass::class);
         $container->addCompilerPass(new WorkflowGuardListenerPass());
+        $container->addCompilerPass(new ResettableServicePass('kernel.reset'), PassConfig::TYPE_AFTER_REMOVING);
```

Comment:
'kernel.reset' should be the default value of the constructor arg instead (as done in other passes)